### ISSUE TITLE
Platform 7.0 does not run on Java 11

### ISSUE THREAD

**Issue by sukhova** [cuba-platform/cuba#1220]
### Environment

- Platform version: 7.0
- Operating system: Windows

### Description of the bug or enhancement

Logs after Refapp's start:

```2018-08-28 10:12:20.384 ERROR [main] eclipselink.metadata - Class java.util.UUID was compiled with unsupported JDK. Report this error to the EclipseLink team, please.
2018-08-28 10:12:20.399 ERROR [main] eclipselink.metadata - Class java.lang.Object was compiled with unsupported JDK. Report this error to the EclipseLink team, please.
2018-08-28 10:12:20.399 ERROR [main] eclipselink.metadata - Class java.lang.Object was compiled with unsupported JDK. Report this error to the EclipseLink team, please.
2018-08-28 10:12:20.399 ERROR [main] eclipselink.metadata - Class java.lang.String was compiled with unsupported JDK. Report this error to the EclipseLink team, please.```

**Comment by andreysubbotin** [comment]
EclipseLink doesn't support on the Java 11. 
See https://github.com/eclipse-ee4j/eclipselink/issues/217 and https://bugs.eclipse.org/bugs/show_bug.cgi?id=538371

**Comment by andreysubbotin** [comment]
Fixed in EclipseLink version 2.7.3-1-cuba
